client/src/Components/SingularPost.js

```diff
@@ -91,7 +105,7 @@ export default function SingularPost(props) {
             <div className={classes.likePost}>
               <Favorite style={{ color: "red", verticalAlign: "middle" }} />
               <h3 style={{ display: "inline-flex", verticalAlign: "middle" }}>
-                &nbsp; {Data.likes} Likes
+                &nbsp; {props.likes} Likes
```

Comment:
Some of this data would come from the Profile component as props, but the comments and likes itself would come the state of this component

Reply:
Absolutely!

Reply:
Though that specific likes is for the Post not the Comment, so it does come from props.

Reply:
yes, that's true